sqlalchemy_bigquery/base.py

```diff
@@ -258,9 +263,16 @@ def visit_column(
             name = self._truncated_identifier("colident", name)
 
         if add_to_result_map is not None:
-            add_to_result_map(name, orig_name, (column, name, column.key), column.type)
+            targets = (column, name, column.key) + result_map_targets
+            if getattr(column, "_tq_label", None):
+                # _tq_label was added in SQLAlchemy 1.4
+                targets += (column._tq_label,)
+
+            add_to_result_map(name, orig_name, targets, column.type)
 
         if is_literal:
+            # note we are not currently accommodating for
+            # literal_column(quoted_name('ident', True)) here
```

Comment:
Is there an issue open for addressing that gap?

Reply:
That comment is part of the copy from sqlalchemy. I'll delete it. :/

Reply:
done